echopype/convert/set_groups_azfp.py

```diff
@@ -167,18 +168,21 @@ def set_platform(self) -> xr.Dataset:
         """Set the Platform group."""
         platform_dict = {"platform_name": "", "platform_type": "", "platform_code_ICES": ""}
         unpacked_data = self.parser_obj.unpacked_data
+        time2 = self.parser_obj.ping_time
+        if np.isnan(unpacked_data["pressure"]).all():
+            depth = [np.nan] * len(time2)
+            water_level = [np.nan]
+        else:
+            depth = -1 * depth_from_pressure(
```

Comment:
`depth` is a positive value (0 at the surface). It's the variable `vertical_offset` that is set to -1 * depth. So, to avoid confusion, let's rename the variable `depth` to `vertical_offset` here, above in the if block, and below in the `vertical_offset` dataarray assignment.

Reply:
Updated!